llvm/include/llvm/SYCLLowerIR/DeviceConfigFile.td

```diff
@@ -188,8 +188,14 @@ class IntelTargetInfo<string Name, list<Aspect> Aspects, list<int> subGroupSizes
 : TargetInfo<Name, IntelBaseAspects # Aspects, subGroupSizesList>;
 // Note: only the "canonical" target names are listed here - see
 // SYCL::gen::resolveGenDevice().
-def : IntelTargetInfo<"intel_gpu_bmg_g21", Fp16Fp64Atomic64, Sg16_32>;
+//
+// TODO: instructions on how to get "???" values below are required. Currently
```

Comment:
@jzc can you please add instructions here? If they are too big, they can be moved to sycl/doc/developer folder and referenced here.

Reply:
The easiest way is to run `sycl-ls --verbose`, but that requires you having the HW. Until you know the data, you can always register a target as _not_ supporting anything.

Reply:
@jzc could you please add a doc and info about these new architectures in the separate upcoming PR?